pkg/output/printer.go

```diff
@@ -142,11 +138,19 @@ func newPagerWithDefault(c *cli.Context) (io.Writer, func()) {
 	output := OutputOption(outputFlag)
 
 	var defaultPager string
-	if output == Table {
+
+	switch output {
```

Comment:
Why is the pager dependent on output format?

Reply:
the pager is selected according to the following priorities:

- the `--pager` flag if set
- pager explicitly set in code when printing items, could be passed for example here https://github.com/temporalio/tctl/blob/50344970f391c5724cd9cb565deae466ce55eec1/cli/namespace_commands.go#L383
- **only if ^ both are not set**, it determines the pager based on the output format

Reply:
for example 

- for tables (the default when printing workflow executions, events) it sets `less`
- for json because it tends to take plenty of lines it sets `more`
- when `namespace describe` or `config show-env` it outputs into stdout (with this PR, used to be `more`)

Reply:
I would always default to stdout instead of having implicit non-portable defaults (seems like Windows does have more BTW). Some commands might default to using a pager but it should be boolean based on the command (or maybe the output) IMO.

Reply:
hmm yea i think i'm good making stdout always be the default and require the tctl dev to explicitly set `PrintOptions.Pager` if they want to set a default. User option would override them

I would like to make it a separate PR though as it will also require to make tctl changes in all places that print something. Should be easier for reviews

Reply:
I'd like to think what the best behavior would be here.
Is using pager dependent on command or output type?
Should it be on by default?
Not blocking the PR though.

Reply:
thinking of this more and i'm leaning towards always falling back to `stdout` by default (instead of falling back to determining the pager based on the output type (table/json)

this means to switch from (ordered by priority) :

1. if user passed `--pager` (`$PAGER`) => use user's pager

2. if dev passed 
```
po := &output.PrintOptions{ Pager: pager.Less }
output.PrintItems(c, items, po)
```
=> use dev's pager. This allows to specify what default behavior you want for a command/specific data

3. fall back to a pager based on the output type. If "table" -> use less , if "json" -> use more

to
1. same as above
2. same as above
3. always fall back to `stdout`

this makes dev's to explicitly tell they want to use "less" or other pager by default (ie. unless user picked something else)